test/e2e/kubectl/kubectl.go

```diff
@@ -1405,7 +1405,7 @@ metadata:
 			Testname: Kubectl, version
 			Description: The command ‘kubectl version’ MUST return the major, minor versions,  GitCommit, etc of the Client and the Server that the kubectl is configured to connect to.
 		*/
-		framework.ConformanceIt("should check is all data is printed ", func() {
+		framework.ConformanceIt("should check is all data is printed [Arktos-CI]", func() {
```

Comment:
same here

Reply:
Changed to another two more essential cases.